src/service/ActivityService.ts

```diff
@@ -124,6 +124,9 @@ ActivityService.get("/participant/:participant_id/activity", async (req: Request
     const TypeRepository = repo.getTypeRepository()
     const ActivityRepository = repo.getActivityRepository()
     let participant_id = req.params.participant_id
+    //check for inclusion of binary data     
+    let ignore_binary:boolean=!!req.query.ignore_binary?(req.query.ignore_binary=="true"?true:false):false
```

Comment:
Can you please make sure you are using `eslint` and `prettier` to format your code correctly?

Reply:
Yes. We used to it. Please ignore this pull request
and check [https://github.com/BIDMCDigitalPsychiatry/LAMP-server/pull/60](https://github.com/BIDMCDigitalPsychiatry/LAMP-server/pull/60)